io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties

```diff
@@ -3,18 +3,16 @@ io.add=Operation Adds subsystem
 io.remove=Operation Removes subsystem
 io.worker=Defines workers
 io.worker.add=Adds new worker
-io.worker.remove=Removes workers
-io.worker.task-max-threads=task-max-threads
-io.worker.read-threads=read-threads
-io.worker.task-core-threads=task-core-threads
-io.worker.stack-size=stack-size
-io.worker.task-limit=task-limit
-io.worker.io-threads=Number of IO threads to use
-io.worker.thread-daemon=thread-daemon
-io.worker.task-keepalive=task-keepalive
+io.worker.remove=Removes worker
+io.worker.task-max-threads="Specify the maximum number of threads for the worker task thread pool.\
+  If not set, default value used which is calculated by formula cpuCount * 16"
+io.worker.stack-size=The stack size (in bytes) to attempt to use for worker threads.
+io.worker.io-threads="Specify the number of I/O threads to create for the worker.  \
+  If not specified, a default will be chosen, which is calculated by cpuCount * 2"
+io.worker.task-keepalive=Specify the number of milliseconds to keep non-core task threads alive.
 io.buffer-pool=Defines buffer pool
 io.buffer-pool.add=Adds new buffer pool
-io.buffer-pool.remove=Removes buffer -pol
-io.buffer-pool.buffers-per-slice=How many buffers per slice
-io.buffer-pool.buffer-size=How big is the buffer
-io.buffer-pool.direct-buffers=Does the buffer pool use direct buffers
+io.buffer-pool.remove=Removes buffer ppol
+io.buffer-pool.buffers-per-slice=How many buffers per slice, if not set value optimal buffer is calculated avaliable RAM resources in your system.
+io.buffer-pool.buffer-size=The size of each buffer slice in bytes, if not set value optimal buffer is calculated avaliable RAM resources in your system.
```

Comment:
s/value optimal buffer/optimal value/g
s/avaliable/based on available/g   (note the typo in 'avaliable')